lib/rspec/rails/configuration.rb

```diff
@@ -84,23 +84,25 @@ def self.initialize_configuration(config)
       # but requires this workaround:
       config.add_setting :rendering_views, :default => false
 
-      def config.render_views=(val)
-        self.rendering_views = val
-      end
+      config.instance_exec do
```

Comment:
YARD issues warnings about writing the methods directly on the object:

> [warn]: in YARD::Handlers::Ruby::MethodHandler: Undocumentable method defined on object instance
> [warn]:         in file 'lib/rspec/rails/configuration.rb':73:
>
>         73: def config.render_views=(val)

Using `instance_exec` provides access to the same singleton scope allowing us to write the methods. Another option would be place all of those methods into a module and then access the singleton class directly (providing proper 1.8.7 backwards compat) to `extend` aforementioned module.